cmd/wego/main.go

```diff
@@ -13,13 +13,54 @@ import (
 	fluxBin "github.com/weaveworks/weave-gitops/pkg/flux"
 )
 
+var VERSION = "0.0.0-dev.0"
+
 var options struct {
 	verbose bool
 }
 
 var rootCmd = &cobra.Command{
-	Use:   "wego",
-	Short: "Weave GitOps",
+	Use:           "wego [subcommand]",
+	Version:       VERSION,
+	SilenceUsage:  true,
+	SilenceErrors: true,
+	Short:         "Weave GitOps",
+	Long:          "Command line utility for managing Kubernetes applications via GitOps.",
+	Example:`
+  # Get verbose output for any wego command
+  wego [command] -v, --verbose
+
+  # Get wego app help
+  wego help app
+
+  # Add application to wego control from a local git repository
+  wego app add
```

Comment:
I think this example and the next should have backslashes after each line except the last:
```
wego app add \
      --path ./podinfo \
      --name podinfo

Reply:
The example listed above has been changed to:
```
  # Add application to wego control from a local git repository
  wego app add . --name <myapp>
  OR
  wego app add <myapp-directory>
```